internal/app/frontend/frontend_service_test.go

```diff
@@ -79,6 +81,83 @@ func TestDoCreateTickets(t *testing.T) {
 	}
 }
 
+func TestDoGetAssignments(t *testing.T) {
+	testTicket := &pb.Ticket{
+		Id: "test-id",
+	}
+
+	totalAssignments := 2
+	var wantAssignments []*pb.Assignment
+
+	for i := 0; i < totalAssignments; i++ {
+		wantAssignments = append(wantAssignments, &pb.Assignment{Connection: string(i)})
+	}
+
+	tests := []struct {
+		description        string
+		preAction          func(*testing.T, statestore.Service)
+		senderGenerator    func(chan *pb.Assignment, *int) func(*pb.Assignment) error
+		wantCode           codes.Code
+		wantAssignmentChan chan *pb.Assignment
```

Comment:
You may not have this at all here. you are simply initializing it here but then really using it to get assignments - so right before executing the test case (before the call to doGetAssignments) you can just initialize a gotAssignmentChannel channel variable. (set it to length of wantAssignments described in other comment)